### PR TITLE
Map Point Tool Toggle - Issue #159

### PR DESCRIPTION
**Pull request by csmoore**
Issue #159 - Allow the Map Point Tool to Toggle & revert to last tool used.

### CODE REVIEW

File: source/Visibility/ProAppVisibilityModule/ViewModels/ProLOSBaseViewModel.cs [hunk 1]
```diff
@@ -109,7 +138,34 @@ public bool IsRunning
...
+                {
+                    ObserverToolActive = false;
+                    TargetToolActive = true;
+                }
+                else
+                {
+                    ObserverToolActive = false;
+                    TargetToolActive = false;
+
+                    DeactivateTool("ProAppVisibilityModule_MapTool");
```
Comment (kgonzago) [inline source/Visibility/ProAppVisibilityModule/ViewModels/ProLOSBaseViewModel.cs]: @csmoore may want to convert string this to a const

File: source/Visibility/ArcMapAddinVisibility/ViewModels/TabBaseViewModel.cs [hunk 2]
```diff
@@ -37,18 +37,49 @@ public TabBaseViewModel()
...
+        {
+
+            // remove graphics from map
+        }
+
+        protected virtual void OnActiveToolChanged(object obj)
+        {
+            string currentActiveToolName = obj as string;
+
+            if ((currentActiveToolName == "Esri_ArcMapAddinVisibility_MapPointTool"))
```
Comment (kgonzago) [inline source/Visibility/ArcMapAddinVisibility/ViewModels/TabBaseViewModel.cs]: @csmoore convert string to a const

File: source/Visibility/ProAppVisibilityModule/ViewModels/ProLOSBaseViewModel.cs (hunk 1 above)
Comment (csmoore) [inline source/Visibility/ProAppVisibilityModule/ViewModels/ProLOSBaseViewModel.cs]: Sure - will fix

**Comment by csmoore** [comment]
@kgonzago - fixed multiple+repeated ID strings in https://github.com/Esri/visibility-addin-dotnet/pull/176/commits/ab8d70ed08b18aa05ac8bf867552af2c4ed83ac0 - good catch (especially since this was already defined in ArcMap addin config.designer.cs, couldn't find something similar in Pro though)

**Pull request merged**